src/main/java/com/uber/cadence/serviceclient/exceptions/StickyWorkerUnavailableException.java

```diff
@@ -0,0 +1,4 @@
+package com.uber.cadence.serviceclient.exceptions;
+
+public class StickyWorkerUnavailableException extends ServiceClientException {
```

Comment:
This is unused and I don't think can actually propagate to the client

Reply:
deleted